### ISSUE TITLE
The Docker BuildStrategy isn't supported for OpenShift Online

### ISSUE THREAD

**Issue by willb** [ophicleide/ophicleide.github.io#4]
We're currently using this and thus should investigate alternatives, like ~custom or~ s2i.

https://docs.openshift.org/latest/admin_guide/securing_builds.html

**Comment by elmiko** [comment]
would this take the form of converting ophicleide-training into an s2i or other repo based build?

**Comment by willb** [comment]
@elmiko yeah, s2i is the way to go.  I guess custom (unsurprisingly?) isn't available either.

**Comment by elmiko** [comment]
ack, should we just migrate this to a card on the -training repo?
